stream_alert/alert_processor/outputs.py

```diff
@@ -803,3 +805,272 @@ def dispatch(self, **kwargs):
                                  Payload=alert_string)
 
         return self._log_status(resp)
+
+@output
+class JiraOutput(StreamOutputBase):
+    """JiraOutput handles all alert dispatching for Jira"""
+    __service__ = 'jira'
+
+    LOGIN = 'rest/auth/1/session'
```

Comment:
Can you maybe add a suffix to these so their usage throughout the class is more clear? Maybe `LOGIN_ENDPOINT`, etc